dependencies.yaml

```diff
@@ -161,13 +161,15 @@ dependencies:
         matrices:
             - matrix:
                 cuda: "12.*"
+                cuda_suffixed: "true"
               packages:
                 - librmm-cu12==24.8.*,>=0.0.0a0
             - matrix:
                 cuda: "11.*"
+                cuda_suffixed: "true"
               packages:
                 - librmm-cu11==24.8.*,>=0.0.0a0
-            - matrix:
+            - matrix: null
```

Comment:
It might be nice to be consistent about when we do and don't use inline tables (some of the other PRs for this did), but nbd.

Reply:
I guess in all those cases the package list only contained YAML anchors and not actual package specs.

Reply:
yeah exactly, that was the difference here.`>` inside a `[]` causes a YAML-parsing error, and when I surrounded this in single quotes that caused some of the pattern-matching in `ci/release/update-version.sh` to break. I thought just splitting it out to an indented map was simpler and safer than modifying `update-version.sh` to account for the possibility of quotes.